notebooks/name-entity-extraction.ipynb

```diff
@@ -0,0 +1,452 @@
+{
```

Comment:
<p>Can you update that screenshot to one that corresponds to the final app of this notebook? (with the medical stuff)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/graphcore/optimum-graphcore-fork/pull/17/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
<p>I don't think you committed the image?</p>